docs/spec_draft.md

```diff
@@ -850,6 +851,42 @@ implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.exponential_minus_one
+
+### Semantics
+
+Performs element-wise exponential minus one operation on `operand` tensor and
```

Comment:
"exponential operation" is well defined? Or we can write
```Performs element-wise exponentiation of Euler's number `e` by the `operand` tensor followed by a decrement of 1 and produces a `result` tensor.```

Reply:
Given we explain the operation, we might(or might not) choose to omit the stuff about IEEE-754 and corner cases.

Reply:
Exponential operations is well defined and also in stablehlo.exponential. I like the current way since it is more consistent with stablehlo.exponential's semantics and is also concise. Wdyt about keeping the semantics as-is?